deployment/aws-k8s/example-runner/deployServicesToK8s.sh

```diff
@@ -44,7 +44,7 @@ fi
 
 # Begin script in case all parameters are correct
 helm dep up
-helm upgrade --install palisade . \
+helm upgrade --install --wait --atomic palisade . \
```

Comment:
Are we really sure we want atomic?
Problem is if it fails there is no way to get any debug or log info out. Then you have to remove the atomic and re-run hoping to get the same issue.
The argument against this is you potentially leave pods thrashing.

Reply:
still outstanding